src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -203,23 +192,25 @@ private function getHelperListBridge(): FeatureHelperListBridge
     /**
      * @return void
      */
-    private function setToolbarActions(): void
+    private function setHeaderToolbarActions(): void
     {
         $controllerConfiguration = $this->getControllerConfiguration();
         $index = $controllerConfiguration->legacyCurrentIndex;
         $token = $controllerConfiguration->token;
 
-        $controllerConfiguration->addToolbarAction(new HeaderToolbarAction('new_feature', [
-            //@todo: replace by $this->generateUrl('admin_features_add') when creation is fully migrated
-            'href' => $index . '&addfeature&token=' . $token,
-            'desc' => $this->trans('Add new feature', 'Admin.Catalog.Feature'),
-            'icon' => 'process-icon-new',
-        ]));
-        $controllerConfiguration->addToolbarAction(new HeaderToolbarAction('new_feature_value', [
-            'href' => $index . '&addfeature_value&id_feature=' . (int) Tools::getValue('id_feature') . '&token=' . $token,
-            'desc' => $this->trans('Add new feature value', 'Admin.Catalog.Help'),
-            'icon' => 'process-icon-new',
-        ]));
+        $controllerConfiguration
+            ->addHeaderToolbarAction('new_feature', [
+                //@todo: replace by $this->generateUrl('admin_features_add') when creation is fully migrated
```

Comment:
The `admin_features_add` already exists (end the edit as well `admin_features_edit`) Maybe we should uncomment it and use it right now. Since the horizontal list is destined to work with those vertical forms anyway (we just need to keep the `_legacy_link` commented for now to prevent automatic generation)